python/cugraph/tests/dask/test_hibench_small.py

```diff
@@ -16,6 +17,7 @@
     import networkx as nx
 
 
+@pytest.mark.skip(reason="ongoing dask issue")
```

Comment:
Isn't this a part of the previously merged PR (https://github.com/rapidsai/cugraph/pull/477/files)? Why does this appear again?

Reply:
Since this branched off from before that PR was merged, I had the option to merge it or add this line for the tests to pass. I could undo this and rebase the whole PR.

Reply:
Not a big deal as long as this gets properly merged.